module.json

```diff
@@ -3,9 +3,9 @@
     "name": "Wiki",
     "description": "Collect and share information with your very own Wiki.",
     "keywords": ["wiki"],
-    "version": "1.3.14",
+    "version": "1.3.15",
```

Comment:
We should also increase here at least to `1.4.0`, since the minVersion was increased.

Reply:
done in commit https://github.com/humhub/humhub-modules-wiki/commit/389db5c7a715d3e8fc0a8f83f6be4bfd9d2fc201